README.md

```diff
@@ -0,0 +1,47 @@
+# Bankruptcy Oversight Support Systems (BOSS)
+
+## [United States Trustee Program](https://www.justice.gov/ust)
+
+The United States Trustee Program is the component of the Department of Justice responsible for overseeing the administration of bankruptcy cases and private trustees under 28 U.S.C. § 586 and 11 U.S.C. § 101, et seq. We are a national program with broad administrative, regulatory, and litigation/enforcement authorities whose mission is to promote the integrity and efficiency of the bankruptcy system for the benefit of all stakeholders–debtors, creditors, and the public. The USTP consists of an Executive Office in Washington, DC, and 21 regions with 90 field office locations nationwide.
+
+[Learn More](https://www.justice.gov/ust/about-program)
+
+## Applications
+
+### Frontend
+
+BOSS is a React application which acts as the main place for oversight work to take place.
+
+Note that any commands listed in this section should be run from the `boss` directory.
+
+#### Requirements
+
+Node version 18.13.0 or above.
+
+#### Running
+
+If you have never run the application before, major changes have been made since you last ran, or your `node_modules` folder has been deleted, you will first need to run the following command to install dependencies:
+
+```shell
+npm install
+```
+
+To run the application directly, execute...
+
+```shell
+npm start
+```
+
+This will serve the client on port 3000.
+
+# Contributing
+
+## Style and Secrets Enforcement
+
+[pre-commit](https://pre-commit.com) allows us to make use of hooks that enforce style and prevent secrets from being committed to the repo. As such, it is expected that all code pushed to the repository have had these pre-commit hooks executed on all files created or updated. The configuration can be seen in `.pre-commit-config.yaml`.
```

Comment:
Took a bit, but I managed to get `pre-commit` working locally. Should have instructions on getting this installed and running. For me, it took a few steps:
1. Open admin powershell prompt, and do `pip install pre-commit` (may even want to ensure folks have python & pip installed first in readme)
2. Then I had to run both `pre-commit autoupdate` and then `pre-commit install`
3. Updated the `.pre-commit-config.yaml` to include typescript files, which are not by default.
4. Added some pws and the commit failed locally (as expected). Can also verify with a `pre-commit run --all-files`

Reply:
@governmentSponsored I'd rather not duplicate install instructions that exist on the linked pre-commit site, especially considering `pip` is only one way that it can be installed. But I do think it makes sense to explicitly state that pre-commit must be installed.

Reply:
@jamesobrooks that's fair, linking to install instructions and indicating it needs to be installed works for me